pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -53,14 +54,11 @@ sub generate_advice {
 
     my $cpanel_version = Cpanel::Version::getversionnumber();
 
-    my $rpm = Cpanel::RPM->new();
-    $self->{has_clam} = $rpm->has_rpm('cpanel-clamav');
-
     eval {
 
         # These checks will only run on v80 and higher
         if (   Cpanel::Version::compare( $cpanel_version, '>=', $IMUNIFY360_MINIMUM_CPWHM_VERSION )
-            && require Whostmgr::Imunify360
+            && _can_load_module('Whostmgr::Imunify360')
```

Comment:
Hopefully this is just an extra precaution and not a bug fix... This was already supposed to be taken care of by the version comparison directly above it.

Reply:
The checks were like this before I touched it. I just didn't like the require's and wanted to change them.